tools/project_manager/page_detail.php

```diff
@@ -43,18 +47,55 @@
     $round_for_page_selection = null;
 }
 
-$project = new Project($projectid);
 $state = $project->state;
 $title = $project->nameofwork;
 $page_details_str = _('Page Detail');
 
 output_header("$page_details_str: $title", NO_STATSBAR);
 
+echo "<div style='display: flex; justify-content: space-between'>";
```

Comment:
Is this necessary? We don't seem to use anything like this the other places where we output a linkbox.

Reply:
I didn't want to special case it, but the float gets positioned very badly because of the wide page details table if we do it the same way that linkbox is done on the other pages.

I went through a few iterations to end up with this solution. NB we do use flex boxes in a few other parts of the codebase.

Reply:
Ah yes, the "page is stupid wide with a lot of rounds" thing.

Let's add a comment here on why we're doing it this way for the next person that comes along.

Reply:
Done.